### DISCUSSION TITLE
Is this the future of lxdware going forward ?

### DISCUSSION THREAD

**Discussion by ithakaa** [PenningLabs/lxconsole#8] [General]
Can we expect that this will be the platform that your main development efforts will focus on ?

**Comment by matthewalanpenning** [comment]
Yes. Starting in October I am planning to allocate a lot of time to start pushing this software to version 1.0, following the [roadmap](https://github.com/PenningLabs/lxconsole/blob/main/roadmap.txt) I have in the code. Once I get this to a stable version I will starting directing people to use this platform.
